examples/website/contour/app.js

```diff
@@ -0,0 +1,130 @@
+import React, {PureComponent, Fragment} from 'react';
+import {render} from 'react-dom';
+import {StaticMap} from 'react-map-gl';
+import {ContourLayer} from '@deck.gl/aggregation-layers';
+import DeckGL from '@deck.gl/react';
+import RangeSlider from './rangeSlider';
+
+// Set your mapbox token here
+const MAPBOX_TOKEN = process.env.MapboxAccessToken; // eslint-disable-line
+
+// Source data CSV
+const DATA_URL =
+  'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/3d-heatmap/heatmap-data.csv'; // eslint-disable-line
+
+const INITIAL_VIEW_STATE = {
+  longitude: -1.4157267858730052,
+  latitude: 52.232395363869415,
+  zoom: 6.6,
+  minZoom: 5,
+  maxZoom: 15,
+  pitch: 40.5,
+  bearing: -27.396674584323023
+};
+
+const THRESHOLD = {
+  MIN: 1,
+  MAX: 20
```

Comment:
MAX value is very less, depending on cell size, weight of cell could be more than 20. Ideally it should be max 32 bit float, but I understand that makes slider too sensitive. Lets make it 500 and see.

Reply:
I tried 500 value for max threshold but it is too high that other isoband are not visible. I am updating to 50.

Reply:
All the thresholds you are setting are non overlapping, changing it to 500, should only affect one of the contours.

Reply:
Done